maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+from typing import Any, Dict, Iterable, List, Optional
+
+from mistralai.models import ChatCompletionResponse, CompletionEvent, CompletionChunk
+
+from ..logger import GenerationRequestMessage
+
+
+class MistralUtils:
+    @staticmethod
+    def parse_message_param(
+        messages: Optional[Iterable[Any]],
+    ) -> List[GenerationRequestMessage]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)